rust/libnewsboat-ffi/src/utils.rs

```diff
@@ -228,6 +228,11 @@ pub extern "C" fn rs_get_random_value(rs_max: u32) -> u32 {
     abort_on_panic(|| utils::get_random_value(rs_max))
 }
 
+#[no_mangle]
+pub extern "C" fn rs_get_auth_method(method: &str) -> u64 {
```

Comment:
FFI functions only accept C types. In this case, `*const c_char` instead of `&str`. See `rs_quote_if_necessary` above for an example on how to convert the C type into a Rust string. A comment on `unwrap()` is mandatory, and should explain why that particular call won't panic under normal conditions.

Reply:
Thanks for reminding me that, I will send in the changes later. But which `unwrap()` do you mean?

Reply:
Oh, my bad, you won't need an `unwrap` here.